src/frontend/src/lib/services/fiat.services.ts

```diff
@@ -0,0 +1,39 @@
+import { ETHEREUM_TOKEN_ID } from '$lib/constants/tokens.constants';
+import { Erc20PriceByContract, EthPrice } from '$lib/providers/coingecko.providers';
+import { balancesStore } from '$lib/stores/balances.store';
+import { fiatStore } from '$lib/stores/fiat.store';
+import { toastsError } from '$lib/stores/toasts.store';
+import { get } from 'svelte/store';
+
+export const loadFiatBalance = async (): Promise<{ success: boolean }> => {
+	const balances = get(balancesStore);
+
+	// https://github.com/microsoft/TypeScript/issues/1863
+	// @ts-ignore
```

Comment:
remove this ts-ignore and handle the issue, I don't think there is a good reason to not too here